packages/@dcl/inspector/src/components/Tree/Tree.tsx

```diff
@@ -28,6 +28,8 @@ type Props<T> = {
   onAddChild: (value: T, label: string) => void
   onRemove: (value: T) => void
   onToggle: (value: T, isOpen: boolean) => void
+  getDragValue?: (value: T) => unknown
```

Comment:
mm I think this here is dangerous. It changes the type of `value` to be `unknown`, but the drop handler types it as `T` and will handle it as such. That could cause some runtime errors if the actual type of `value` is not compatible with the new `unknown` type, here for example: https://github.com/decentraland/js-sdk-toolchain/blob/main/packages/%40dcl/inspector/src/components/Tree/Tree.tsx#L86